en/docs/chapter_sorting/counting_sort.md

```diff
@@ -71,14 +71,14 @@ The implementation code of counting sort is shown below:
 
 ## Algorithm characteristics
 
-- **Time complexity is $O(n + m)$, non-adaptive sort**: Involves traversing `nums` and `counter`, both using linear time. Generally, $n \gg m$, and the time complexity tends towards $O(n)$.
-- **Space complexity is $O(n + m)$, non-in-place sort**: Utilizes arrays `res` and `counter` of lengths $n$ and $m$ respectively.
+- **Time complexity is $O(n + m)$, non-adaptive sort**: It involves traversing `nums` and `counter`, both using linear time. Generally, $n \gg m$, and the time complexity tends towards $O(n)$.
+- **Space complexity is $O(n + m)$, non-in-place sort**: It uses array`res` of lengths $n$ and array `counter` of length $m$ respectively.
```

Comment:
please add a space between "It uses array" and "`res`"

Reply:
done